AK/NumberFormat.cpp

```diff
@@ -25,6 +25,23 @@ DeprecatedString human_readable_size(u64 size)
     return DeprecatedString::formatted("{:.1f} EiB", static_cast<float>(size) / static_cast<float>(EiB));
 }
 
+DeprecatedString human_readable_quantity(u64 quantity)
+{
+    if (quantity < 1 * KiB)
```

Comment:
If you want to use power of ten, don't use `KiB` which is in power of two.

Reply:
I wanted to use powers of two though, using powers of ten was a option in command line utilities that could be added later

Is it correct to use these constants in this case where the quantity in question is not in bytes but you still want to comparee things with 1024, 1024*1024 etc?